### PR TITLE
Added support for printing better log information

### PR DESCRIPTION
**Pull request by vbaddi**
Summary:

Added helper function to log error information from
OnnxModelLoader, Caffe2ModelLoader and CommonOperatorLoader

Documentation: None

Test Plan: Ninja Test is run.

### CODE REVIEW

**Comment by vbaddi** [comment]
@jfix71 could you please review this change?

**Comment by vbaddi** [comment]
Hi @mciprian13 ,

As suggested in [#4762](https://github.com/pytorch/glow/pull/4762) added the changes separately with helper function for ONNXModelLoader, Caffe2ModelLoader and CommonOperatorLoader.
Could you please review this change?

Thank you

**Comment by mciprian13** [comment]
@vbaddi Looks good to me! Although I am not in the position of making an official review. If I understand correctly this PR replaces the previous PR #4762?

**Comment by vbaddi** [comment]
@mciprian13, Thank you for the comment.
Sure., once this is approved.
Ill close the previous PR #4762.

**Comment by vbaddi** [comment]
@mortzur  could you please review this change?

**Comment by vbaddi** [comment]
@jfix71 could you please review this changes and provide review comments if any?

**Comment by jfix71** [comment]
Really sorry for the major delays here, I've been super swamped. I will definitely get to reviewing next week.

**Review by jfix71 — APPROVED** [review]
This is great! Thanks so much @vbaddi

File: include/glow/Importer/CommonOperatorLoader.h [hunk 1]
```diff
@@ -1230,17 +1279,28 @@ class CommonOperatorLoader : public ProtobufLoader {
...
+    RETURN_ERR_IF_NOT(
+        ranges.dims().size() == 3,
+        opErrMsg(op, strFormat("GatherRanges: Ranges must be a 3D vector, but "
+                               "found vector size %zu ",
+                               ranges.dims().size())));
     RETURN_ERR_IF_NOT(ranges.dims()[2] == 2,
-                      "Last dimension of ranges must be 2.");
+                      opErrMsg(op, strFormat("GatherRanges: Last dimension of "
+                                             "ranges must be 2, but found %lu",
+                                             ranges.dims()[2])));
```
Comment (jfix71) [inline include/glow/Importer/CommonOperatorLoader.h:1298]: @vbaddi There's a legit CI issue here when we use 32 bit mode -- can you fix? Probably just need to cast.

**Review by jfix71 — CHANGES_REQUESTED** [review]

File: include/glow/Importer/CommonOperatorLoader.h (hunk 1 above)
Comment (vbaddi) [reply include/glow/Importer/CommonOperatorLoader.h]: Done.

**Review by jfix71 — APPROVED** [review]

**Pull request closed without merging**